templates/webpack.angular.js

```diff
@@ -96,6 +97,13 @@ module.exports = env => {
         additionalLazyModuleResources: additionalLazyModuleResources
     });
 
+    let sourceMapFilename = "[file].map";
```

Comment:
Maybe it'll be better to extract this function to common place and reuse it in all config files.